app/pages/reservation/reservation-information/InternalReservationForm.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import PropTypes from 'prop-types';
+
+import injectT from '../../../i18n/injectT';
+import FormTypes from '../../../constants/FormTypes';
+
+class UnconnectedInternalReservationForm extends Component {
+  render() {
+    const {
+      t,
+      commentsMaxLengths,
+      valid
+    } = this.props;
+    return (
+      <div className="app-ReservationDetails">
+        <h2 className="app-ReservationPage__title">{t('ReservationForm.premiseStaffOnly')}</h2>
+        <Row>
+          <Col md={1}>
+            <label className="app-InternalReservationForm__checkbox">
+              <Field
+                component="input"
+                id="internalReservationChecked"
+                name="internalReservation"
+                type="checkbox"
+              />
+              <span className="custom-checkmark" />
+            </label>
+          </Col>
+          <Col md={11}>
+            <span className="app-ReservationDetails__value">
+              {t('ReservationForm.internalReservation')}
+              <br />
+              {t('ReservationForm.internalReservationDescription')}
+            </span>
+          </Col>
+        </Row>
+        <Row>
+          <Col md={1}>
+            <label className="app-InternalReservationForm__checkbox">
+              <Field
+                component="input"
+                name="markAsClosed"
+                type="checkbox"
+              />
+              <span className="custom-checkmark" />
```

Comment:
This is hacky by the way.
We dont implement checkbox like this.

What you need to do is: 
- Have the Field as checkbox as you did.
- Give the vanilla HTML checkbox styling as design show. 
- *No* extra custom-checkmark just for decorating purpose.

Reply:
This hacky html is removed.